assets/styles/viper.css

```diff
@@ -1,5 +1,7 @@
 body {
   background-color: white;
+  width: 100%;
```

Comment:
This problem not repro in my environment (windows/linux with chrome/firefox). Could you try to look for a reason? Are there any javascript errors there?

My recommendation is to trigage it as minor bug and accept the CR to don't stack commits.